bin/reth/src/utils.rs

```diff
@@ -26,6 +26,11 @@ use std::{
 };
 use tracing::info;
 
+/// Exposing the `reth_db` crate
+pub mod utils {
```

Comment:
I was not quite sure where to put this, but seemed like the function needed to be exposed.

Reply:
agree, but I feel like this should be called `db`